src/PhpSpreadsheet/Calculation/Calculation.php

```diff
@@ -2205,7 +2205,7 @@ class Calculation
         ],
         'SIN' => [
             'category' => Category::CATEGORY_MATH_AND_TRIG,
-            'functionCall' => [MathTrig::class, 'builtinSIN'],
+            'functionCall' => [MathTrig\Sin::class, 'funcSin'],
```

Comment:
not `evaluate()`?

Reply:
Sin was not part of this PR. I just noticed that I had neglected to update the function table for Sin when I did implement its replacement. Since I am planning a follow-up PR to change funcWhatever, I would prefer to defer this suggestion until then.